kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/DMNServicesClientImpl.java

```diff
@@ -111,7 +111,11 @@ public DMNServicesClientImpl(KieServicesConfiguration config, ClassLoader classL
             return evaluateDecisions(containerId, payload);
         }
 
-        private ServiceResponse<DMNResult> evaluateDecisions(String containerId, DMNContextKS payload) {
+        /**
+         * Please notice this method is NOT exposed to the API interface.
+         */
+        // DO NOT ADD @Override
+        public ServiceResponse<DMNResult> evaluateDecisions(String containerId, DMNContextKS payload) {
```

Comment:
moved to `pubilc` anyway an instance to the end-user is normally the API interface, which is used by a provisioning mechanism of the general kie-server API.